app/views/providers/means/check_income_answers/_applicant_income_assessment.html.erb

```diff
@@ -5,89 +5,35 @@
-               individual: @legal_aid_application.applicant,
-               state_benefits: @legal_aid_application.applicant.state_benefits,
-               version: "means",
-               read_only:) %>
-  <% end %>
+  <%= render("shared/check_answers/benefits_income", partner: false, read_only:) if @legal_aid_application.client_uploading_bank_statements? %>
 
-  <%= render(
-        "shared/check_answers/payments_summary",
-        payment_type: t(".credits-section-heading"),
-        url: bank_statement_upload ? providers_legal_aid_application_means_regular_incomes_path(@legal_aid_application) : providers_legal_aid_application_means_identify_types_of_income_path(@legal_aid_application),
-        transaction_types: @credit_transaction_types,
-        individual: "Applicant",
-        read_only:,
-      ) %>
+  <!--  Income payments-->
+  <%= render("shared/check_answers/payments_summary", credit: true, partner: false, read_only:) %>
 
   <% if bank_statement_upload %>
-    <%= render(
-          "shared/check_answers/cash_payments",
-          url: providers_legal_aid_application_means_cash_income_path,
-          individual: "Applicant",
-          individual_with_determiner: t("generic.client"),
-          type: :income,
-          read_only:,
-        ) %>
+    <%= render("shared/check_answers/cash_payments", partner: false, type: :income, read_only:) %>
   <% end %>
 
-  <%= render(
-        "shared/check_answers/student_finance",
-        individual: "Client",
-        individual_with_determiner: t("generic.client"),
-        model: @legal_aid_application.applicant,
-        url: providers_legal_aid_application_means_student_finance_path(@legal_aid_application),
-        read_only:,
-      ) %>
+  <%= render("shared/check_answers/student_finance", partner: false, read_only:) %>
 
   <h2 class="govuk-heading-l"><%= t(".outgoings-heading") %></h2>
 
-  <%= render(
-        "shared/check_answers/payments_summary",
-        payment_type: t(".debits-section-heading"),
-        url: bank_statement_upload ? providers_legal_aid_application_means_regular_outgoings_path(@legal_aid_application) : providers_legal_aid_application_means_identify_types_of_outgoing_path(@legal_aid_application),
-        transaction_types: @debit_transaction_types,
-        individual: "Applicant",
-        read_only:,
-      ) %>
+  <!--  Outgoing payments-->
+  <%= render("shared/check_answers/payments_summary", credit: false, partner: false, read_only:) %>
 
-  <% if bank_statement_upload %>
-    <%= render(
-          "shared/check_answers/cash_payments",
-          url: providers_legal_aid_application_means_cash_outgoing_path,
-          individual: "Applicant",
-          individual_with_determiner: t("generic.client"),
-          type: :outgoings,
-          read_only:,
-        ) %>
-  <% end %>
+  <%= render("shared/check_answers/cash_payments", partner: false, type: :outgoings, read_only:) if bank_statement_upload %>
```

Comment:
Nit: Is it possible to have an Inline or Block pattern? Line 27 uses a conditional block, this line uses the condition inline.